src/sentry/search/events/datasets/profile_functions.py

```diff
@@ -291,20 +291,47 @@ def function_converter(self) -> Mapping[str, SnQLFunction]:
                 SnQLFunction(
                     "examples",
                     snql_aggregate=lambda _, alias: Function(
-                        "arrayMap",
+                        # The worst may collide with one of the examples, so make sure to filter it out.
+                        "arrayDistinct",
                         [
-                            # TODO: should this transform be moved to snuba?
-                            Lambda(
-                                ["x"],
-                                Function(
-                                    "replaceAll", [Function("toString", [Identifier("x")]), "-", ""]
-                                ),
-                            ),
                             Function(
-                                "arrayPushFront",
+                                "arrayFilter",
                                 [
-                                    Function("groupUniqArrayMerge(5)", [SnQLColumn("examples")]),
-                                    Function("argMaxMerge", [SnQLColumn("worst")]),
+                                    # Filter out the profile ids for processed profiles
+                                    Lambda(
+                                        ["x"],
+                                        Function(
+                                            "notEquals",
+                                            [Identifier("x"), "00000000000000000000000000000000"],
```

Comment:
nit: should we make this a constant?

Reply:
Changed to `uuid.UUID(int=0).hex` which returns the same thing.